src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -970,7 +970,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
     // the type symbol of the owner of the member that supplies the result value
     lazy val resultSymbol = {
       val sym = lineRep.resolvePathToSymbol(fullAccessPath)
-      // plow through the INSTANCE member when -Yrepl-class-based
+      // plough through the INSTANCE member when -Yrepl-class-based
```

Comment:
is this a US vs UK thing? I would write "plow"

it's fine either way though :-)